scene/animation/animation_mixer.cpp

```diff
@@ -1980,6 +1980,7 @@ void AnimationMixer::reset() {
 }
 
 void AnimationMixer::restore(const Ref<AnimatedValuesBackup> &p_backup) {
+	ERR_FAIL_NULL(p_backup);
```

Comment:
```suggestion
	ERR_FAIL_COND(!p_backup.is_valid());
```
We don't use null checks with `Ref`

Reply:
Oh thanks!